wiki/Ranking_Criteria/id.md

```diff
@@ -215,6 +213,7 @@ Kategori ini berisi perbolehan eksplisit mengenai konsep dan peraturan yang bias
 
 ### Pedoman
 
+- **Berkas audio dan berkas hitsound dari sebuah beatmap tidak boleh menampilkan distorsi suara terdengar dan tanpa sebab**, seperti *clipping*, *muffling*, atau *crackling* yang jelas-jelas tidak dimaksudkan oleh artis atau bukan bagian dari identitas lagu. Penentuan tentang ini paling baik dengan mendengarkan audio, daripada menggunakan perangkat lunak.
```

Comment:
curious, are there any Indonesian words for the terms you italicized? maybe what I am really asking is "would Indonesian mappers understand what this means with respect to audio quality?"

Reply:
I think there is, *clipping* can be `audio terpotong`, *muffling* can be `audio terlalu kecil/teredam`, and leave *crackling* be since there isn't any good word for that.

Reply:
usually, translations to Indonesia are not much correct to understand in general. Since most of it are spreading to another meaning.

Reply:
audio clipping isn't "terpotong" (as in "audio cut") per se tho iirc, it's more like "dikeraskan secara berlebihan sehingga suara-suara tertentu menjadi terbenam" if any

but yeah these are just some example of these technical terms that -- while they *can* be translated -- translating them would only cause the opposite effect of confusing the reader even more (unless of helping them to understand)

Reply:
i think at the very least you can do something like this tho :

```suggestion
- **Berkas audio dan hitsound yang dimiliki oleh beatmap tidak boleh mengandung hal-hal yang dapat berdampak buruk bagi kualitas suara yang ada**, seperti volume yang terlampau keras (*clipping*), volume yang terlampau sunyi (*muffling*), atau bunyi kertakan (*crackling*) yang jelas-jelas tidak dimaksudkan oleh pihak pembuat lagu serta bukan merupakan bagian dari lagu yang bersangkutan.
```